notebooks/community/model_garden/model_garden_pytorch_mistral_peft_tuning.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    # Tranformers provides all API for downloading and working with pre-trained models that are in the HF hub.
</pre><p>Tranformers &gt; Transformers</p><p>huggeringface &gt; Huggeringface</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2645/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>